recipes/ics/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "ics" %}
+{% set version = "0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/C4ptainCrunch/ics.py/archive/v{{ version }}.tar.gz
+  sha256: 48c637e5eb8dfc817b1f3f6b3f662ba19cfcc25f8f71eb42f5d07e6f2c573994
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.6
+    - 'arrow <0.15,>=0.11'
```

Comment:
You can need the quotes.

Reply:
Also, you probably don't need this whole build section, only `host`.

Reply:
Did you try removing the build section here? I'm pretty sure you don't need it. Only host would suffice.

Reply:
Yes, I removed them in the latest commits. The one you're referring to is outdated.

Reply:
Nope. They are still there. I removed them in https://github.com/conda-forge/staged-recipes/pull/13465/commits/4e1181b75a13d86fef68ba6c677bf1ece424721e